pom.xml

```diff
@@ -2144,6 +2153,17 @@
                         </dependency>
                     </dependencies>
                 </plugin>
+
+                <plugin>
+                    <groupId>com.github.eirslett</groupId>
+                    <artifactId>frontend-maven-plugin</artifactId>
+                    <version>${maven.plugin.frontend.version}</version>
+                    <configuration>
+                        <nodeDownloadRoot>https://npmmirror.com/mirrors/node/</nodeDownloadRoot>
```

Comment:
This mirror contains all released versions, not like Apache mirror which only contains the "current version".

Reply:
Should we  set the official download address by [default](https://nodejs.org/dist/) and give a profile to change download source.

Reply:
OK, moved to `mirror-cdn`, you can verify by w/ and w/o `-Pmirror-cdn`

Reply:
I'm not sure, only set to `properties` can be noticed by frontend-maven-plugin's configuration ?

Reply:
You can clear the local cache and do package, the log will show the download link

Reply:
Yeah, i have tested locally. W/ `-Pmirror-cdn`, see log output as expected.
```
Downloading https://npmmirror.com/mirrors/node/v16.19.1/node-v16.19.1-darwin-x64.tar.gz to 
Downloading https://registry.npmmirror.com/pnpm/-/pnpm-7.27.1.tgz to
```